.angular-cli.json

```diff
@@ -1,57 +1,54 @@
 {
```

Comment:
Can you change this file to only have the changes you made to the file? I think we might need to look at some cross editor formatting of code 🤔